typing/typecore.ml

```diff
@@ -148,6 +148,8 @@ type error =
   | No_value_clauses
   | Exception_pattern_disallowed
   | Mixed_value_and_exception_patterns_under_guard
+  | Effect_pattern_below_toplevel
+  | Invalid_continuation_pattern
```

Comment:
These two errors are no longer / not yet triggered, since currently there is no syntax and no specific typing rules for effect handlers.

Reply:
Removed in https://github.com/ocaml/ocaml/pull/10831/commits/6aa1379ba5a4891e4eb76ad6baafcb02ecd616a8.